src/server/game/Motd/MotdMgr.cpp

```diff
@@ -37,69 +39,123 @@ MotdMgr* MotdMgr::instance()
+
+    // Create all world packages after loading motd and localized texts
+    CreateWorldPackages();
 }
 
-void MotdMgr::LoadMotd()
+char const* MotdMgr::GetMotd(LocaleConstant locale)
 {
-    uint32 oldMSTime = getMSTime();
+    // Return localized motd if available, otherwise fallback to enUS
+    auto it = MotdMap.find(locale);
+    if (it != MotdMap.end())
+        return it->second.c_str();
 
-    uint32 realmId = sConfigMgr->GetOption<int32>("RealmID", 0);
+    return MotdMap[LOCALE_enUS].c_str();  // Fallback to enUS if locale is not found
+}
+
+WorldPacket const* MotdMgr::GetMotdPacket(LocaleConstant locale)
+{
+    // Return localized packet if available, otherwise fallback to enUS
+    auto it = MotdPackets.find(locale);
+    if (it != MotdPackets.end())
+        return &it->second;
+
+    return &MotdPackets[LOCALE_enUS];  // Fallback to enUS if locale is not found
+}
+
+std::string MotdMgr::LoadDefaultMotd(uint32 realmId)
+{
     LoginDatabasePreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_MOTD);
     stmt->SetData(0, realmId);
     PreparedQueryResult result = LoginDatabase.Query(stmt);
-    std::string motd;
 
     if (result)
     {
         Field* fields = result->Fetch();
-        motd = fields[0].Get<std::string>();
-    }
-    else
-    {
-        LOG_WARN("server.loading", ">> Loaded 0 motd definitions. DB table `motd` is empty for this realm!");
-        LOG_INFO("server.loading", " ");
+        return fields[0].Get<std::string>();  // Return the main motd if found
     }
 
-    motd = /* fctlsup << //0x338// "63"+"cx""d2"+"1e""dd"+"cx""ds"+"ce""dd"+"ce""7D"+ << */ motd
-        /*"d3"+"ce"*/ + "@|" + "cf" +/*"as"+"k4"*/"fF" + "F4" +/*"d5"+"f3"*/"A2" + "DT"/*"F4"+"Az"*/ + "hi" + "s "
-        /*"fd"+"hy"*/ + "se" + "rv" +/*"nh"+"k3"*/"er" + " r" +/*"x1"+"A2"*/"un" + "s "/*"F2"+"Ay"*/ + "on" + " Az"
-        /*"xs"+"5n"*/ + "er" + "ot" +/*"xs"+"A2"*/"hC" + "or" +/*"a4"+"f3"*/"e|" + "r "/*"f2"+"A2"*/ + "|c" + "ff"
-        /*"5g"+"A2"*/ + "3C" + "E7" +/*"k5"+"AX"*/"FF" + "ww" +/*"sx"+"Gj"*/"w." + "az"/*"a1"+"vf"*/ + "er" + "ot"
-        /*"ds"+"sx"*/ + "hc" + "or" +/*"F4"+"k5"*/"e." + "or" +/*"po"+"xs"*/"g|r"/*"F4"+"p2"+"o4"+"A2"+"i2"*/;;
-    MotdMgr::SetMotd(motd);
+    return ""; // Return empty string if no motd found
+}
+
+void MotdMgr::SetDefaultMotd(std::string& motd)
+{
+    // Set a default motd text only for enUS if no motd is found
+    motd = "@|cffF4A2DThi server runs on Azeroth Core|cff3CE7FFwww.azerothcore.org|r";
+    MotdMap[LOCALE_enUS] = motd;
 
-    LOG_INFO("server.loading", ">> Loaded Motd Definitions in {} ms", GetMSTimeDiffToNow(oldMSTime));
+    // Log that no motd was found and a default is being used for enUS
+    LOG_WARN("server.loading", ">> Loaded 0 motd definitions. DB table `motd` is empty for this realm!");
     LOG_INFO("server.loading", " ");
 }
 
-char const* MotdMgr::GetMotd()
-{
-    return FormattedMotd.c_str();
+void MotdMgr::LoadLocalizedMotds(uint32 realmId) {
```

Comment:
so, since we removed the connection between the tables in the database. We need a check in load that there is a "base" MOTD in the first table that corresponds with the entry in the locale table.

Also bracket on newline

Reply:
Sorry, I dont follow. The base motd is always the en_US one, at least that is what was agreed on before. Or are you referring to the realmIds -1 etc? Would you mind providing more details ? Thank you